ruff.toml

```diff
@@ -0,0 +1,25 @@
+# Support Python 3.10+.
+target-version = "py310"
+# Set the maximum line length to 79.
+fix = true
+line-length = 120
```

Comment:
Let's leave the line length to default for now. It's a bigger conversation if we want to go to 120 and wouold need to reformat everything. I am open to this (79 is quite annoying)

Reply:
Fair point, changed it back to 79